components/centraldashboard/app/routes.ts

```diff
@@ -0,0 +1,30 @@
+import express from 'express';
+import {resolve} from 'path';
+
+import {KubernetesService} from './k8s_service';
+
+const frontEnd: string = resolve(__dirname, 'public');
+
+/**
+ * Routes defined for the Express server.
+ */
+export function routes(k8sService: KubernetesService): express.Router {
+  const routes: express.Router = express.Router();
+
+  routes.use(express.json());
+  routes.use(express.static(frontEnd));
+  routes.get(
```

Comment:
Can we do
```js
routes.use('/api', apiRoutes)
```
And move all `/api` routes to a `apis.ts` file

Reply:
Given the simplicity of the routes at this point and the fact that nothing is under test, I didn't factor it out too broadly. If I were to refactor this I would probably leave the static handling and json middleware in the main server module. What are your thoughts on that?

Reply:
I agree with that approach wholeheartedly. Sounds good, we can wait for post-NEXT to do those changes.